lib/KalturaFTPServer.js

```diff
@@ -276,99 +307,117 @@ FtpSession.prototype = {
 	},
 
 	createReadStream: function () {
+		var err = 'createReadStream: Unimplemented usage';
+		return console.error(err);
 	},
 
-	unlink: function () {
+	unlink: function (path, callback) {
+		var err = 'unlink: Unimplemented usage';
+		callback(err, []);
```

Comment:
For all callbacks, if there is an error and callback is not defined, throw the error.